gradlew.bat

```diff
@@ -1,89 +1,89 @@
-@rem

```

Comment:
Do not add this file (valid for all the `gradlew` files).

Reply:
oh yes must have got in there during my earlier commit! sorry! will remove it asap

Reply:
Just to clarify: you don't need to remove the file, but just to amend the changes 👍

Reply:
As @cortinico mentioned, we shouldn't be deleting these files, just undo your changes.. and similarly for the other gradlew.bat files. We probably shouldn't be touching those. @cortinico to verify

Reply:
sure will reinstatie them